.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -13,7 +13,7 @@ assignees: ''
 
 ---
 **Contributor Resources**
-- [Meshery documentation site](https://meshery.layer5.io/docs/)
-- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
-- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
+- [Meshery documentation site](https://docs.meshery.io/)
```

Comment:
@sayantan1413 lets not change this

Reply:
Okay so @kumarabd I will not change the links for the meshery documentation right?

Reply:
yes, just the GitHub endpoint needs to be updated.